modules/nodes-pods-autoscaling-custom-creating-workload.adoc

```diff
@@ -113,8 +113,8 @@ spec:
           stabilizationWindowSeconds: 300
           policies:
           - type: Percent
-            value: 100
-            periodSeconds: 15
+            value: '100'
+            periodSeconds: '15'
```

Comment:
please revert this, the quotoes are only needed for values in `spec.triggers.metadata`